### ISSUE TITLE
Title

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#7]
@mrshirts @jpotoff @msoroush

I have thrown out a lot of different titles for the manuscript (although although most of them are pretty similar). I would appreciate if we could each vote on our favorite title and/or propose new ones.

Here are the top ones so far:

Histogram-free reweighting to estimate vapor-liquid coexistence properties of non-simulated force fields
Using histogram-free reweighting to estimate vapor-liquid coexistence properties of non-simulated force fields
Histogram-free reweighting enables the estimation of vapor-liquid coexistence properties for non-simulated force fields
A histogram-free reweighting approach to estimate vapor-liquid coexistence properties for non-simulated force fields
A histogram-free reweighting approach for estimating vapor-liquid coexistence properties of non-simulated force fields
Estimating vapor-liquid coexistence properties of non-simulated force fields with histogram-free reweighting

**Comment by msoroush** [comment]
My vote is

A histogram-free reweighting approach to estimate vapor-liquid coexistence properties for non-simulated force fields

Sincerely,
Mohammad
----------------------
Potoff Group - ChE Dept - Wayne State Univ.
E: m.soroush@wayne.edu
C: 313.980.8605
W: 313.577.3758
HTTP://gomc.eng.wayne.edu

________________________________

**Comment by mrshirts** [comment]
Does JCED have any rules about titles not starting with articles?

**Comment by ramess101** [comment]
@mrshirts 

Not sure. This is all it says on Author Guidelines, "Use a specific rather than a general title. The title should be as brief and informative as possible." But scanning through published JCED articles, I did not see a single manuscript with a title that started with an article. So maybe we should avoid it.

**Comment by ramess101** [comment]
To modify the one @msoroush voted for we could say:

Histogram-free reweighting as an approach to estimate vapor-liquid coexistence properties for non-simulated force fields

**Comment by jpotoff** [comment]
@ramess101 @msoroush trying to come up with something better that "non-simulated"...

**Comment by ramess101** [comment]
@jpotoff 

Yeah, I don't love that either. I tried "multiple", but then it sounds like Hamiltonian scaling.

**Comment by jpotoff** [comment]
@ramess101 @msoroush  Not sure about this one, but it's another idea.  
Histogram-free prediction of vapor-liquid coexistence properties for a range of force field parameters from the simulation of a single parameter set.

**Comment by ramess101** [comment]
@jpotoff 

That is certainly a more informative title, although I think it violates the other JCED suggestion that titles be brief. 

Also, one of the NIST reviewers argued that we should not use the term "prediction" when referring to MBAR, since it is really just a post-processing approach that calculates vapor-liquid coexistence values from simulation results. I can see both sides of the argument though.

**Comment by jpotoff** [comment]
@ramess101  Another attempt: histogram-free calculation of vapor-liquid equilibria from Grand canonical Monte Carlo simulations.

**Comment by mrshirts** [comment]
Histogram-free estimation of multiple vapor-liquid equilibria using Grand Canonical Monte Carlo Simulations and reweighting?

**Comment by ramess101** [comment]
Estimation (or Calculation) of multiple vapor-liquid equilibria using Grand canonical Monte Carlo simulations and histogram-free reweighting?

**Comment by jpotoff** [comment]
@ramess101 I think you've got a winner with the last one.  I prefer "calculation" over "estimation", though.

**Comment by ramess101** [comment]
@jpotoff Glad you like it, but I am still not satisfied. What about:

Vapor-liquid equilibria (or coexistence) calculations of multiple force fields (or potentials, Hamiltonians, potential models) using Grand canonical Monte Carlo simulations and histogram-free reweighting

**Comment by ramess101** [comment]
@jpotoff @mrshirts @msoroush 

Is it important that the title mention Grand canonical Monte Carlo?

**Comment by ramess101** [comment]
Some other ideas:

Histogram-free reweighting: A tool to enable post-simulation force field parameterization

Histogram-free reweighting: A tool to calculate vapor-liquid equilibria post-simulation for multiple force fields

**Comment by jpotoff** [comment]
@ramess101 How about this one?  I have a preference for "method" over "tool" because to me a "tool" implies a software product, and we're not writing a software paper.

Histogram-free reweighting: A method enabling post-simulation force field optimization.

**Comment by ramess101** [comment]
@jpotoff I have no preference towards tool over method. 

I like what you have, but I think we could rearrange it a little more to get:

Histogram-free reweighting: A post-simulation force field optimization method (approach)

@mrshirts @msoroush What are your feelings on the last few titles?

Also, @jpotoff any reason to use optimization over parameterization? Just curious because I go back and forth on these two a lot. To me, optimization implies that these are truly the optimal force field parameters (when they are probably near-optimal at best) while parameterization simply means that we are assigning parameters (could be without a scoring function altogether). So I feel like parameterization is a more conservative word but it could also sell MBAR short because even a random number generator could parameterize a force field (although it certainly would not optimize the force field). Anyways, I am being fairly pedantic here.

**Comment by ramess101** [comment]
@jpotoff @mrshirts @msoroush Any of these titles look good or spark some ideas?

Post-simulation force field optimization to vapor-liquid equilibria with histogram-free reweighting

Post-simulation force field optimization with histogram-free reweighting vapor-liquid coexistence calculations

Post-simulation force field optimization with histogram-free Grand canonical Monte Carlo vapor-liquid coexistence calculations

**Comment by jpotoff** [comment]
@ramess101 I like the word "optimization" because with this method we can say definitively that one is not going to find a better set of parameters for the Mie/LJ potential.  That's what you're showing with the heat maps.  I'm ok with "parameterization" though if you think "optimization" is too strong, though.  

Another version of words with friends:
_Histogram-free reweighting: A post-simulation approach for force field optimization (parameterization)_ 

=> my thought is this is probably the shortest descriptive title we can come up with.  We can leave out grand canonical Monte Carlo because even though that is the simulation method we are using, one could apply this method to other types of simulation, as you've done with ITIC and molecular dynamics.

The other one I like is this one:
_Calculation of multiple vapor-liquid equilibria using Grand canonical Monte Carlo simulations and histogram-free reweighting_

**Comment by msoroush** [comment]
I like it. With minor modification:

Histogram-free reweighting: A post-simulation approach to optimize force field

Sincerely,
Mohammad

**Comment by jpotoff** [comment]
@msoroush @ramess101 one more tweak:

_Histogram-free reweighting: A post-simulation approach to optimize force field parameters_
Or the more wordy:
_Histogram-free reweighting: A post-simulation approach for the optimization of force field parameters_

This one is also pretty good: _Histogram-free reweighting: A post-simulation approach to force field optimization_

**Comment by ramess101** [comment]
@msoroush @jpotoff 

I think we are almost there. We could run with about any of those, but I was wondering, do we want to make it clear that these are non-bonded parameters? E.g.,

Histogram-free reweighting: A post-simulation approach to optimize non-bonded (intermolecular) parameters

**Comment by jpotoff** [comment]
_Histogram-free reweighting: A post-simulation approach to optimize non-bonded parameters_

I'm good with this one.  I can see the importance of clarifying non-bonded parameters, since we are not parameterizing the complete for field (bond, angles, torsions).

**Comment by ramess101** [comment]
@jpotoff @msoroush 

Yeah, I think that could be important. I guess the only thing that is lost here that could be important is the notion of phase equilibria. Is there a good way to include that in the title?

Histogram-free reweighting: A post-simulation approach to optimize non-bonded parameters with multiple (rapid) vapor-liquid coexistence (equilibria) calculations

**Comment by msoroush** [comment]
I like it:
A post-simulation approach to optimize non-bonded parameters with rapid vapor-liquid equilibria calculations

**Comment by ramess101** [comment]
@jpotoff @msoroush 

I know this approach could be applied to other force fields, but we could also promote the MiPPE force field effectively through the title which would cover both the non-bondeds and VLE:

Histogram-free reweighting: A post-simulation approach to optimize Mie potentials for phase equilibria

**Comment by msoroush** [comment]
> Histogram-free reweighting: A post-simulation approach to optimize Mie potentials for phase equilibria

You have my vote!

**Comment by ramess101** [comment]
@mrshirts 

I agree with Michael that somewhere along the way we lost the novelty of this paper, namely, applying MBAR to GCMC. So I like Michael's title, after correcting GEMC to GCMC :-)

Histogram-free reweighting with Grand canonical Monte Carlo: A post-simulation approach to optimize non-bonded potentials for phase equilibria

Not exactly succinct though

**Comment by mrshirts** [comment]
Hah, sorry.  Wrong ensemble :) Too focused on writing a final.

**Comment by ramess101** [comment]
@jpotoff @mrshirts @msoroush 

Here are the JCED requirements that are specific to molecular modeling simulation studies:

![image](https://user-images.githubusercontent.com/23408516/49950323-d5d6e980-feb4-11e8-80ac-0fc8c3f9a0d8.png)

Should we mention cyclohexane specifically since we have new parameters/data?

**Comment by msoroush** [comment]
@ramess101 
We studied many compounds, branched alkane, alkyne, and cyclohexane. I think it is better to not mentioning cyclohexane.

**Comment by ramess101** [comment]
@msoroush

True. I agree that we dont want to list all the different compounds,
especially if our title is referring more to the method than the force
field and the data.

However, we only apply the post-simulation optimization to cyclohexane. So
we only have new MiPPE parameters for cyclohexane. Although you could argue
that epsilon scaling produced new parameters, we never really report those.
And we only really have “new” data for cyclohexane, since the other data
were more for validation.

I guess I just don’t want the MiPPE cyclohexane parameters to be lost if
someone is searching for them. Having this in the title would make sure
people see this as part of the MiPPE series.

@jpotoff @mrshirts

What do you think about this title?

Histogram-free reweighting with Grand canonical Monte Carlo: A
post-simulation optimization of the cyclohexane Mie potential for phase
equilibria

**Comment by jpotoff** [comment]
@ramess101 Trying another way:  Mie Potentials for Phase Equilibria: Parameterization of cyclohexane using histogram-free reweighting grand canonical Monte Carlo simulations .

**Comment by jpotoff** [comment]
@ramess101 I'm ok with leaving cyclohexane out and shrinking the title.  I'm just wordsmithing on some of the ideas that are being posted.

**Comment by ramess101** [comment]
@jpotoff

I appreciate it. All ideas are welcome. I am not opposed to this morphing
into a MiPPE paper, I just didnt write it with that in mind. I think the
cyclohexane parameterization is certainly the show pony for this
manuscript, so thats why I kind of wanted it mentioned in the title. But I
dont want that the overshadow the methodology that is the novel aspect.

**Comment by mrshirts** [comment]
But I don't want that the overshadow the methodology that is the novel
aspect.

Yeah, I'd agree on this.

**Comment by jpotoff** [comment]
@mrshirts @ramess101: "Histogram-free reweighting with grand canonical Monte Carlo: A post-simulation approach to optimize non-bonded potentials for phase equilibria."  Good enough?

**Comment by ramess101** [comment]
@jpotoff @mrshirts 

Yeah I think this is the best title: it focuses on GCMC-MBAR and it does not limit the method to just Mie potentials or just cyclohexane.

One small question that JCED might end up dictating anyways, should the capitalization of GCMC be "Grand canonical Monte Carlo", "grand canonical Monte Carlo" or "Grand Canonical Monte Carlo"?

**Comment by jpotoff** [comment]
Typically only "Monte Carlo" is capitalized.

**Comment by ramess101** [comment]
@jpotoff @mrshirts

Sorry for my obsession with the title, but I was thinking about how to make
it a little more succinct. What do you think about eliminating “a” and
“approach”:

Histogram-free reweighting with grand canonical Monte Carlo:
Post-simulation optimization of non-bonded potentials for phase equilibria

**Comment by mrshirts** [comment]
Titles are important (My greatest creative achievement is probably the Science perspective article title on Folding@Home - "Screensavers of the world, unite!").  

I think that latest title is works well.

**Comment by ramess101** [comment]
@mrshirts

Haha, that is an amazing title! I dont think I’m aiming for that level of
creativity...maybe in our next publication together.

@jpotoff Any objections to the latest title proposal?

**Comment by jpotoff** [comment]
@ramess101 @mrshirts 

> Histogram-free reweighting with grand canonical Monte Carlo:
> Post-simulation optimization of non-bonded potentials for phase equilibria

I'm good with this.

**Comment by ramess101** [comment]
@jpotoff @mrshirts 

OK, I think we finally have a winner
